### ISSUE TITLE
default file creation permissions may cause filebeat not to start up

### ISSUE THREAD

**Issue by mmguero** [cisagov/Malcolm#24]
User [_imp0ster on reddit](https://www.reddit.com/r/netsec/comments/bzfv2f/malcolm_is_a_powerful_easily_deployable_network/er3h48f/) discovered this bug that happens on some systems

```
        $ docker-compose logs -f filebeat
        ...
        filebeat_1       | 2019-06-12 17:56:56,442 INFO success: filebeat entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
        filebeat_1       | 2019-06-12 17:56:56,444 INFO exited: filebeat (exit status 1; not expected)
        filebeat_1       | 2019-06-12 17:56:57,448 INFO spawned: 'filebeat' with pid 390
        filebeat_1       | Exiting: error loading config file: config file ("filebeat.yml") can only be writable by the owner but the permissions are "-rw-rw-r--" (to fix the permissions use: 'chmod go-w /usr/share/filebeat/filebeat.yml')
        filebeat_1       | 2019-06-12 17:56:57,491 INFO success: filebeat entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
        filebeat_1       | 2019-06-12 17:56:57,500 INFO exited: filebeat (exit status 1; not expected)
        filebeat_1       | 2019-06-12 17:56:58,504 INFO spawned: 'filebeat' with pid 398
        filebeat_1       | Exiting: error loading config file: config file ("filebeat.yml") can only be writable by the owner but the permissions are "-rw-rw-r--" (to fix the permissions use: 'chmod go-w /usr/share/filebeat/filebeat.yml')
        filebeat_1       | 2019-06-12 17:56:58,629 INFO success: filebeat entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
        filebeat_1       | 2019-06-12 17:56:58,634 INFO exited: filebeat (exit status 1; not expected)
        filebeat_1       | 2019-06-12 17:56:59,638 INFO spawned: 'filebeat' with pid 406
        filebeat_1       | Exiting: error loading config file: config file ("filebeat.yml") can only be writable by the owner but the permissions are "-rw-rw-r--" (to fix the permissions use: 'chmod go-w /usr/share/filebeat/filebeat.yml')
        filebeat_1       | 2019-06-12 17:56:59,695 INFO success: filebeat entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
        filebeat_1       | 2019-06-12 17:56:59,698 INFO exited: filebeat (exit status 1; not expected)
        filebeat_1       | 2019-06-12 17:57:00,733 INFO spawned: 'filebeat' with pid 416
        filebeat_1       | Exiting: error loading config file: config file ("filebeat.yml") can only be writable by the owner but the permissions are "-rw-rw-r--" (to fix the permissions use: 'chmod go-w /usr/share/filebeat/filebeat.yml')
        filebeat_1       | 2019-06-12 17:57:00,863 INFO success: filebeat entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
        filebeat_1       | 2019-06-12 17:57:00,864 INFO exited: filebeat (exit status 1; not expected)
        filebeat_1       | 2019-06-12 17:57:01,868 INFO spawned: 'filebeat' with pid 440
        filebeat_1       | Exiting: error loading config file: config file ("filebeat.yml") can only be writable by the owner but the permissions are "-rw-rw-r--" (to fix the permissions use: 'chmod go-w /usr/share/filebeat/filebeat.yml')
        filebeat_1       | 2019-06-12 17:57:01,900 INFO success: filebeat entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
        filebeat_1       | 2019-06-12 17:57:01,905 INFO exited: filebeat (exit status 1; not expected)
        filebeat_1       | 2019-06-12 17:57:02,908 INFO spawned: 'filebeat' with pid 451
```

I need to fix the permissions on filebeat.yml in the installer and/or startup script to make sure this doesn't happen.

As a workaround.

1. ./scripts/stop.sh
2. chmod go-w ./filebeat/filebeat.yml
3. start things back up

**Comment by 0x-Imp0ster** [comment]
Thanks, this worked for me.

**Comment by mmguero** [comment]
Fixed in release v1.2.1
